proto/cosmos/auth/beta/query.proto

```diff
@@ -32,5 +32,5 @@ message QueryParamsRequest{ }
 
 // QueryParamsResponse is response type for the Query/Params RPC method
```

Comment:
since we're cleaning code, I'd like to suggest adding a `.` at the end of all comments (which I believe is what some people have been starting to do).